server/ERNI.PBA.Server.DataAccess/Model/Request.cs

```diff
@@ -30,6 +31,8 @@ public class Request
 
```

Comment:
Does it make sense to still have the budgetId? We have the list of transactions and we have userId to know, who created the request